lang/cs/Org.Apache.REEF.IMRU.Examples/Run.cs

```diff
@@ -116,25 +140,33 @@ public static void RunBroadcastReduceTest(IConfiguration tcpPortConfig, bool run
                 injector = TangFactory.GetTang()
                     .NewInjector(OnREEFIMRURunTimeConfiguration<int[], int[], int[]>.GetYarnIMRUConfiguration(), tcpPortConfig);
             }
-
-            if (faultTolerant)
+            if (fixedFailures)
             {
-                var broadcastReduceFtExample = injector.GetInstance<PipelinedBroadcastAndReduceWithFaultTolerant>();
-                broadcastReduceFtExample.Run(numNodes - 1, chunkSize, iterations, dims, mapperMemory, updateTaskMemory, maxRetryNumberInRecovery, totalNumberOfForcedFailures);
+                if (faultTolerant)
+                {
+                    var broadcastReduceFtExample = injector.GetInstance<PipelinedBroadcastAndReduceWithFaultTolerant>();
+                    broadcastReduceFtExample.Run(numNodes - 1, chunkSize, iterations, dims, mapperMemory, updateTaskMemory, maxRetryNumberInRecovery, totalNumberOfForcedFailures);
+                }
+                else
+                {
+                    var broadcastReduceExample = injector.GetInstance<PipelinedBroadcastAndReduce>();
+                    broadcastReduceExample.Run(numNodes - 1, chunkSize, iterations, dims, mapperMemory, updateTaskMemory);
+                }
             }
             else
             {
-                var broadcastReduceExample = injector.GetInstance<PipelinedBroadcastAndReduce>();
-                broadcastReduceExample.Run(numNodes - 1, chunkSize, iterations, dims, mapperMemory, updateTaskMemory);
+                var broadcastReduceRandFailureExample = injector.GetInstance<PipelinedBroadcastAndReduceWithRandomFailures>();
+                broadcastReduceRandFailureExample.Run(numNodes - 1, chunkSize, iterations, dims, mapperMemory, updateTaskMemory, maxRetryNumberInRecovery, failureProbability, minTimeOutInSec, expThroughPut);
             }
         }
 
         /// <summary>
         /// Run IMRU examples from command line
         /// </summary>
         /// Sample command line:  
-        /// .\Org.Apache.REEF.IMRU.Examples.exe true 500 8900 1000 broadcastandreduce 20000000 1000000 1024 1024 10
-        /// .\Org.Apache.REEF.IMRU.Examples.exe true 500 8900 1000 broadcastandreduceft 20000000 1000000 1024 1024 100 5 2
+        /// .\Org.Apache.REEF.IMRU.Examples.exe true 500 8900 1000 BroadcastAndReduce 20000000 1000000 1024 1024 10
```

Comment:
We're getting more and more sample command lines here, and the choice becomes confusing... Since we don't have per-argument help, I'd add a brief comment to each line explaining what it does, something like "To run broadcast-and-reduce IMRU job without intentionally introduced failures", "To run broadcast-and-reduce IMRU job with fixed intentionally introduced failures", "To run broadcast-and-reduce IMRU job with randomly introduced failures".